nf_core/modules/lint/main_nf.py

```diff
@@ -346,8 +346,9 @@ def check_process_section(self, lines, registry, fix_version, progress_bar):
         if url is None:
             continue
         try:
+            container_url = "https://" + urlunparse(url) if not url.scheme == "https" else urlunparse(url)
```

Comment:
Just out of curiosity - what is the advantage of defining the container-url outside of the call to request.head? Is it only for "aesthetical" reasons?